index.js

```diff
@@ -1,10 +1,14 @@
+/* eslint-disable no-sync */
 // Import Third-party Depedencies
 import cacache from "cacache";
 import lodashGet from "lodash.get";
 
+import fs from "fs";
+import path from "path";
```

Comment:
Node.js dependencies must be imported first (before third-party). And think to prefix with `node:`

```js
// Import Node.js Dependencies
import fs from "node:fs";
import path from "node:path";
```